pymeasure/instruments/srs/sr850.py

```diff
@@ -0,0 +1,180 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2024 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import time
+from . import sr830
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set, \
+    strict_discrete_range
+
+class SR850(sr830.SR830):
+    EXPANSION_VALUES = None  #[1, 10, 100]
+    EXPANSION_RANGE = [1, 256]
+    INPUT_CONFIGS = ['A', 'A - B', 'I']
+    INPUT_GAINS = ['1 MOhm', '100 MOhm']
+    REFERENCE_SOURCES = ['Internal', 'Sweep', 'External']
+    SNAP_ENUMERATION = {"x": 1, "y": 2, "r": 3, "theta": 4,
+                        "aux in 1": 5, "aux in 2": 6, "aux in 3": 7, "aux in 4": 8,
+                        "frequency": 9, "trace 1": 10, "trace 2": 11, 
+                        'trace 3': 12, 'trace 4': 13} 
+    OUTPUT_INTERFACE = ['RS232', 'GPIB']
+
+    def __init__(self, adapter, name="Stanford Research Systems SR850 Lock-in amplifier",
+                 **kwargs):
+        # write term lf or cr on RS232, lf or eoi on GPIB
+        kwargs.setdefault('write_termination', '\n')  
+        super().__init__(
+            adapter,
+            name,
+            **kwargs
+        )
+        if 'GPIB' in self.name.capitalize():
+            self.output_interface = 'GPIB'
+    output_interface = Instrument.control(
+        "OUTX?", "OUTX %d",
+        """ An string property that controls the output interface. Inputs are

```

Comment:
Docstrings are in imperative mode (at least the first sentence).

For a control, it should start with "Control", for a setting with "Set" and for a measurement with "Get" or "Measure".
For example:
```suggestion
        """Control the output interface. Inputs are

```